learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -313,24 +336,26 @@ Likewise, hold *Control* (*Command* on Mac) to drop a setter:
 Signals
 ~~~~~~~
 
-In the *Connecting Signals to a Function* item we have already learned about *Signals*. It is also possible to create your own
-signals in a script and use them. For this, just do the same steps you did for variables in the previous step, except for *Signals*:
+In the *Connecting Signals to a Function* item we have already learned about *Signals*. It is also possible to create your own signals in a script and use them. For this, just do the same steps you did for variables in the previous step, except for *Signals*:
 
 .. image:: img/visual_script29.png
 
-Signal can also be edited via right click menu to customize arguments:
+
+A Signal can also be edited via right click menu to customize arguments:
 
 .. image:: img/visual_script30.png
 
-The signal you have just created will also show together with the other node signals, this allows to eventually connect to it from another script
-from another *Scene Node*:
+
+The signal you have just created will also show together with the other node signals, this allows you to eventually connect to it from another script from another *Scene Node*:
```

Comment:
just created will appear in the Inspector along with the built-in node signals.  This allows you to connect it from another script in a different *Scene Node*.